package-lock.json

```diff
@@ -0,0 +1,9772 @@
+{
+  "name": "task-app",
+  "version": "0.1.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "task-app",
+      "version": "0.1.0",
+      "hasInstallScript": true,
+      "dependencies": {
+        "@emotion/react": "^11.11.3",
+        "@emotion/styled": "^11.11.0",
+        "@mui/icons-material": "^5.15.15",
+        "@mui/material": "^5.15.6",
+        "@mui/x-date-pickers": "^7.1.1",
+        "@prisma/client": "^5.12.0",
+        "@reduxjs/toolkit": "^2.2.3",
+        "@types/date-fns": "^2.6.0",
+        "@vercel/postgres": "^0.8.0",
+        "copilot-node-sdk": "^2.0.0",
+        "dayjs": "^1.11.10",
+        "http-status": "^1.7.4",
+        "immutability-helper": "^3.1.1",
+        "next": "^14.1.4",
+        "prisma": "^5.12.0",
+        "react": "^18",
+        "react-dnd": "^16.0.1",
+        "react-dnd-html5-backend": "^16.0.1",
+        "react-dom": "^18",
+        "react-redux": "^9.1.0",
```

Comment:
Please remove `package-lock.json`

We have yarn as our package manager. Do not mix npm and yarn lockfiles as they can cause problems resolving dependencies. Our build / deploy also uses yarn so there will be inconsistencies there.